react/features/mobile/navigation/middleware.js

```diff
@@ -29,7 +30,12 @@ function _conferenceFailed({ dispatch }, next, action) {
     // We need to cover the case where knocking participant
     // is rejected from entering the conference
     if (error.name === JitsiConferenceErrors.CONFERENCE_ACCESS_DENIED) {
-        dispatch(appNavigate(undefined));
+        dispatch(openDialog(AlertDialog, {
+            contentKey: {
+                key: 'lobby.joinRejectedMessage'
+            },
+            onSubmit: () => dispatch(appNavigate(undefined))
```

Comment:
Should we use `cancelKnocking` instead? It will clear the knocking state as well.